src/main/java/eu/execom/pomodoroTeam/services/UserDaoImpl.java

```diff
@@ -0,0 +1,30 @@
+package eu.execom.pomodoroTeam.services;
+
+import eu.execom.pomodoroTeam.entities.UserEntity;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityManager;
+
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class UserDaoImpl implements UserDao {
```

Comment:
You already have a `UserRepository` interface where all logic for accessing database table `User` should go. You can use `@Query` annotation to write a similar query as the one below and completely remove this class and interface it implements.

Reply:
Please remove both `UserDao` and `UserDaoImpl` from the code base.